Formula/source-to-image.rb

```diff
@@ -0,0 +1,14 @@
+class SourceToImage < Formula
+  desc "Tool for building source and injecting into docker images"
+  homepage "https://github.com/openshift/source-to-image"
+  url "https://github.com/openshift/source-to-image/releases/download/v1.1.3/source-to-image-v1.1.3-ddb10f1-darwin-amd64.tar.gz"
```

Comment:
As source-to-image is in Go the user will need that on his machine to build it. Could you explain, please, why we can't just download the already built binary?